client/coral-framework/constants/asset.js

```diff
@@ -9,3 +9,5 @@ export const UPDATE_ASSET_SETTINGS_FAILURE = 'UPDATE_ASSET_SETTINGS_FAILURE';
 export const OPEN_COMMENTS = 'OPEN_COMMENTS';
 export const CLOSE_COMMENTS = 'CLOSE_COMMENTS';
 export const UPDATE_COUNT_CACHE = 'UPDATE_COUNT_CACHE';
+
+export const VIEW_ALL_COMMENTS = 'VIEW_ALL_COMMENTS';
```

Comment:
I just added the hook, since I could see us doing something with it eventually. Redux complains if you don't return an action from an action generator http://localhost:3000/assets/title/p#24420e5b-1c1f-4d6b-979b-67a25bddc5b0